src/app/modules/group/components/group-leave/group-leave.component.html

```diff
@@ -11,12 +11,12 @@
   </p>
 
   <div class="button">
-    <alg-button
-      class="p-button-rounded-full"
+    <p-button
+      styleClass="alg-button p-button-rounded-full"
       icon="fa fa-times"
       i18n-label label="Leave this group"
-      (click)="leaveGroup()"
+      (onClick)="leaveGroup()"
       [disabled]="group.isMembershipLocked"
-    ></alg-button>
+    ></p-button>
```

Comment:
The button icon alignment looks broken, can you check if an issue exists for that and if not create one?
![image](https://user-images.githubusercontent.com/16257159/169243542-4367b74f-bd5f-42df-8874-d6068fcf2b17.png)
I spotted the misalignment in other buttons, so I guess it's at the button level:
Manager add button: 
![image](https://user-images.githubusercontent.com/16257159/169244174-a4e6a44c-7f00-4cd5-a1dd-bf1f1f3ec0d6.png)